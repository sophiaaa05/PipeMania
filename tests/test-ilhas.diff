2,4c2,4
< BD	BC	BC	BE
< BD	BB	BB	BE
< VD	VC	VD	VC
\ No newline at end of file
---
> BD	BE  BD  BE
> BD	BE  BD  BE
> VD	VC	VD	VC
