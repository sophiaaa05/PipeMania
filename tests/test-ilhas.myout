VB	VE	VB	VE
BD	BE	BD	BE
BD	BE	BD	BE
VD	VC	VD	VC
