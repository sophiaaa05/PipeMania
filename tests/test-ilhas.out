VB	VE	VB	VE
BD	BC	BC	BE
BD	BB	BB	BE
VD	VC	VD	VC